src/current/v25.1/ui-replication-dashboard.md

```diff
@@ -171,22 +171,6 @@ Metric | Description
 -------|------------
 `{node}` | The rate of `ReplicaUnavailableError` events that have occurred per aggregated interval of time on that node since the `cockroach` process started.
 
-## Paused Follower
```

Comment:
@kvoli and @nhamlani101 since the eng design doc (AIUI) said paused follower is obsolete and not that helpful i deleted from the docs, HOWEVER it appears at least on my beta.3 cluster that this UI graph still exists, so Idk if we should just leave this here in teh docs? or do we need to delete this graph from the UI in the codebase? or ??? please advise

Reply:
We should delete it, so removing it here is a good idea. Thanks for the callout, filed: https://github.com/cockroachdb/cockroach/issues/141425